samples/custom-login/public/index.html

```diff
@@ -0,0 +1,192 @@
+<html>
+  <head>
+    <!-- https://global.oktacdn.com/okta-auth-js/4.0.0/okta-auth-js.polyfill.js -->
```

Comment:
Perhaps a comment to highlight this is the polyfill for IE (or at least link to spot in README)?